src/app/core/shell/toolbar/toolbar.component.html

```diff
@@ -46,34 +51,45 @@
     Separate into its own component in shared module for reuse.
     Implement functionality.
   -->
+
+  <!-- <button #step1g> Hi</button> -->
   <div *ngIf="searchVisible" class="search-bar" fxFlex="auto" fxHide.lt-sm="true" [@fadeInOut]>
     <mat-form-field class="search">
       <mat-label>Search</mat-label>
       <input matInput type="text" (focusout)="toggleSearchVisibility()">
     </mat-form-field>
   </div>
 
-  <button mat-icon-button class="ml-1" (click)="toggleSearchVisibility()" fxHide.lt-sm="true">
-    <fa-icon icon="search" size="lg"></fa-icon>
-  </button>
-
-  <mifosx-language-selector class="ml-1 language" fxHide.lt-md></mifosx-language-selector>
+  <div #globalSearch>
+    <button mat-icon-button class="ml-1" (click)="toggleSearchVisibility()" fxHide.lt-sm="true">
+      <fa-icon icon="search" size="lg"></fa-icon>
+    </button>
+  </div>
 
-  <mifosx-theme-picker fxHide.lt-md></mifosx-theme-picker>
+  <div #languageSelector>
+    <mifosx-language-selector class="ml-1 language" fxHide.lt-md></mifosx-language-selector>
+  </div>
 
+  <div #themePicker>
+    <mifosx-theme-picker fxHide.lt-md></mifosx-theme-picker>
+  </div>
+  
   <!--
     TODO: Notifications
     Separate into its own component in shared module for reuse.
     Implement functionality.
   -->
-  <button mat-icon-button class="ml-1" matTooltip="Notifications">
-    <fa-icon icon="bell" size="lg"></fa-icon>
-  </button>
-
-  <button mat-icon-button class="ml-1 img-button" [matMenuTriggerFor]="applicationMenu">
-    <img src="assets/images/user_placeholder.png">
-  </button>
+  <div #notifications>
+    <button mat-icon-button class="ml-1" matTooltip="Notifications">
+      <fa-icon icon="bell" size="lg"></fa-icon>
+    </button>
+  </div>
```

Comment:
Wouldn't it be better to use ng-containers instead of divs? Currently they may not affect the view, but if any changes are needed in future to UI, prescence of div can affect the view.
![Screenshot from 2020-06-28 19-56-38](https://user-images.githubusercontent.com/54709463/85950308-aa71fd80-b979-11ea-86ec-781706f1bd41.png)

Reply:
Read about ng-containers here https://angular.io/guide/structural-directives